package.xml

```diff
@@ -15,6 +15,8 @@
   <exec_depend>rospy</exec_depend>
   <exec_depend>rosbridge_library</exec_depend>
   <exec_depend>std_msgs</exec_depend>
+  <exec_depend>mosquitto</exec_depend>
+  <!--<exec_depend>mosquitto-clients</exec_depend>-->
```

Comment:
this line is not necessary since `paho-mqtt` works as a mosquitto (mqtt) client.